massa-bootstrap/src/server.rs

```diff
@@ -305,12 +298,60 @@ impl BootstrapServer {
                 }
             }
         }
-        // wait for bootstrap sessions to finish
-        while bootstrap_sessions.next().await.is_some() {}
```

Comment:
As sessions are now spawned into a thread that runs in the background, I _don't think_ this is required, as the execution context (tokio runtime) of the spawned threads remain active.

This removal would need to be revisited if we opt for a design that has a dedicated execution context (might be useful for tracking the number of active sessions, for example).

Reply:
This should only block on the thread hosting the main-loop, and not the binary-wide-system.

Ungraceful shutdowns of active sessions is fine in the short-term.